server/chat-commands/room-settings.ts

```diff
@@ -360,7 +366,11 @@ export const commands: ChatCommands = {
 			buffer += `<p><strong>Single-command permissions:</strong> (will affect one command)<br />`;
 			buffer += `<code>` + permissions.join(`</code> <code>`) + `</code></p>`;
 			buffer += `<p><details class="readmore"><summary><strong>Sub-commands:</strong> (will affect one sub-command, like /roomevents view)</summary>`;
```

Comment:
```suggestion
```

I like how you didn't even notice that you repeated this line. :p

Reply:
Hey, i noticed when I did it locally :c

Reply:
I mean, it's repeated in your screenshot too.

Reply:
yes but i thought i'd prioritize screenshot over removing a line